keda/templates/22-metrics-deployment.yaml

```diff
@@ -86,10 +86,10 @@ spec:
           env:
             - name: WATCH_NAMESPACE
               value: {{ .Values.watchNamespace | quote }}
-            {{- if .Values.http.timeout }}
```

Comment:
this is not needed anymore?

Reply:
I think so because [values file defines a default value](https://github.com/kedacore/charts/blob/main/keda/values.yaml#L325). I mean, that if is always true

Reply:
If you prefer, I can add it again, but I removed to keep templates cleanest as possible